### DISCUSSION TITLE
Reading History Scrobbling?

### DISCUSSION THREAD

**Discussion by ThoughtzThruKeyz** [gotson/komga#1743] [Ideas]
Not 100% sure if this is necessarily feasible, but it would be great if we could scrobble our reading history and or keep track of what's in our library/what we've read, dropped, etc. With the help of Anilist, MAL, or something equivalent.

**Comment by gotson** [comment]
What you want to do is not clear, can you clarify ? Komga keep history of what you have read, but does not integrate with other online services.

There are some 3rd party scripts that can do that for you: https://komga.org/docs/faq/#how-can-i-sync-reading-progress-with-tracker-websites
